src/screens/ConfigurePMTs/PaymentMethodConfigUtils.res

```diff
@@ -0,0 +1,299 @@
+let dropdownClassName = (options: array<SelectBox.dropdownOption>) =>
+  options->Array.length > 5 ? "h-80" : "h-full"
+
+let getAdvanceConfiguration = (
+  advanceConfiguration: option<ConnectorTypes.advancedConfigurationList>,
+): React.element => {
+  let config = switch advanceConfiguration {
+  | Some(obj) => {
+      let firstThree = obj.list->Array.slice(~start=0, ~end=3)->Array.toString
+      let restCount = obj.list->Array.length - 3
+      obj.list->Array.length > 3
+        ? <div>
+            {`${firstThree},`->React.string}
+            <span className="text-blue-811">
+              {`+${Belt.Int.toString(restCount)} more`->React.string}
+            </span>
+          </div>
+        : <div> {firstThree->React.string} </div>
+    }
+  | None => "NA"->React.string
+  }
+  config
+}
+
+let encodeAdvanceConfig = (advanceConfig: option<ConnectorTypes.advancedConfigurationList>) => {
+  switch advanceConfig {
+  | Some(config) =>
+    [
+      ("type", JSON.Encode.string(config.type_)),
+      ("list", JSON.Encode.array(config.list->Array.map(Js.Json.string))),
+    ]
+    ->Dict.fromArray
```

Comment:
->Dict.fromArray
    ->JSON.Encode.object

to ->LogicUtils.getJsonFromArrayOfJson